src/Test.EndToEnd.Baselining/BaseliningTester.cs

```diff
@@ -316,5 +328,11 @@ private static void SortForBaselining(Run run)
             List<Result> results = (List<Result>)run.Results;
             results.Sort(DirectResultMatchingComparer.Instance);
         }
+
+        private static void DeleteAndCreateDirectory(string directoryPath)
+        {
+            if (Directory.Exists(directoryPath)) { Directory.Delete(directoryPath, true); }
```

Comment:
>true [](start = 83, length = 4)

Ultra-nit: Since `bool` parameters aren't self-documenting, I like to use named parameters for them:
```
Directory.Delete(directoryPath, recursive: true);
```